examples/oemof_0.2/excel_reader/dispatch.py

```diff
@@ -33,125 +36,303 @@
 from oemof.tools import logger
 from oemof import solph
 from oemof import outputlib
+from oemof.graph import create_nx_graph
 from matplotlib import pyplot as plt
+import networkx as nx
 
 
-def nodes_from_excel(filename):
+def nodes_from_excel(filename, header_lines=0):
+    """Read node data from Excel sheet
```

Comment:
The option header_lines gives the impression that this function is generic but this is not the way the excel reader works. To avoid misleading signal in this direction I would remove the parameter.